src/components/Form.js

```diff
@@ -593,6 +602,16 @@ class Form extends Component {
             Submit
           </Button>
         </ValidatorForm>
+        <Dialog
+          open={thanksOpen}
+          onClose={this.handleClose}
+        >
+          <DialogContent>
+            <DialogContentText>
+              Thank you for submitting! Your submission should appear on the map within a week.
```

Comment:
I like this this short and to the point message. Katie had added a similar message in her email yesterday to be displayed on form submission. It says "Thank you for your submission! Please note that the system will display your observation on the map after a period of one week". I am okay with either of these messages as the essence of both is the same.

Reply:
Updated!